docs/src/modules/components/JoyUsageDemo.tsx

```diff
@@ -268,20 +268,21 @@ export default function JoyUsageDemo<T extends { [k: string]: any } = {}>({
               );
             }
             if (knob === 'radio') {
+              const labelId = `${componentName}-${String(propName)}`;
               return (
                 <Box key={propName as string}>
                   <Typography
-                    id={`${componentName}-${propName}`}
+                    id={labelId}
                     fontSize="xs"
                     fontWeight="md"
                     sx={{ mb: 1, textTransform: 'capitalize' }}
                   >
-                    {propName}
+                    {String(propName)}
```

Comment:
isn't the `propName` already a string?

Reply:
It is a `Symbol` not text, this is why I needed the cast.

Reply:
Looks like the type of `propName` is too wide, React [doesn't accept symbol porperties](https://github.com/facebook/react/issues/7552). Does it make sense to narrow it down?
```
propName: Extract<keyof ComponentProps, string>;
```